edx/edx-enterprise
# [ENT-172] Catalog Service API graceful degradation

**Description:** Makes Course Catalog API tolerant to empty lists returned from `get_edx_api_data`

**JIRA:** [ENT-172](https://openedx.atlassian.net/browse/ENT-172)

**Testing instructions:**

1. **Do not start Course Catalog**
2. Open "Manage Leaarners View" for any customer.
3. Observe that it didn't crashed with 500 error


**Merge checklist:**

**Post merge:**

**Author concerns:** Only last commit is specific to the issue - rest are problems slipped through code review and/or CI build from previous PRs.
